### ISSUE TITLE
output discovery project id, assistant integration id, watsonx saas project id and instance id

### ISSUE THREAD

**Issue by vburckhardt** [terraform-ibm-modules/terraform-ibm-rag-sample-da#20]

**Comment by vburckhardt** [comment]
essentially surfacing mostly same outputs as the watson saas da for the resources created by the rag da https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/blob/main/outputs.tf

**Comment by rajatagarwal-ibm** [comment]
Just realised that in the RAG DA its referring to a sub-folder in the SAAS DA https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/tree/main/configure_project and that's just outputting project_id. 

The configure_project doesn't call the root module at all... hence won't able to get any of the output mentioned above here -https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/blob/main/outputs.tf

**Comment by rajatagarwal-ibm** [comment]
PR https://github.com/terraform-ibm-modules/terraform-ibm-rag-sample-da/pull/23

**Comment by rajatagarwal-ibm** [comment]
PR merged

**Comment by vburckhardt** [comment]
Hi @rajatagarwal-ibm  - just tested on a live deployement, the URLs are broken - they link to non existing pages. Did they work in your testing?

I notice also some differences between watson saas da outputs, and rag da ouputs. 

For project_url for instance
WatsonDA- https://dataplatform.cloud.ibm.com/projects/7d81d1f1-d88f-4780-9c76-ce94efc1805f?context=wx&sync_account_id=190c293e9fda4c6684b5acf4b17871b8
RAG DA  - https://dataplatform.cloud.ibm.com/projects/0b9987-e7b5-4272-9b7f-da7996f95424 (missing context=wx&sync_account_id=190c293e9fda4c6684b5acf4b17871b8 perhaps?)

**Comment by rajatagarwal-ibm** [comment]
@vburckhardt had a discussion with Andrea who developed SaaS DA. He said he has occasionally encountered this issue where the `project_id` is trimmed from the beginning. But in our case it's trimmed from the end too.

This is the project ID
<img width="883" alt="image" src="https://github.com/terraform-ibm-modules/terraform-ibm-rag-sample-da/assets/108140212/b39d05af-b4e7-4481-860d-05ae1f457fe7">

But the returned value in the output is `3b96dd-ad80-4b8d-befe-d2d5bebbb8a`

Basically if you add `2c` as prefix and `c` as suffix it will be the right project_id `2c3b96dd-ad80-4b8d-befe-d2d5bebbb8ac`.

I have requested Andrea to return whole object as well, so that we too can play around to fetch the right ID until he gets the root cause of the issue.

**Comment by vburckhardt** [comment]
Still some issues with the watson discover + assistant url. Project url is fixed.

**Comment by rajatagarwal-ibm** [comment]
Tracking it here https://github.com/terraform-ibm-modules/terraform-ibm-rag-sample-da/issues/21
